package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nuxtjs/markdownit",
   "version": "2.0.0",
-  "homepage": "nuxt-community/markdownit-module",
+  "homepage": "https://github.com/nuxt-community/markdownit-module",
```

Comment:
I think we need to use "repo" instead of homepage for fixing ☺️

Reply:
@pi0 I deal with the commit☺️ https://github.com/nuxt-community/markdownit-module/pull/33/commits/ac1af32ea849677c42dcebfdd15ef454bbd8716d I appreciate your review, thank you!:3